Orange/widgets/utils/scaling.py

```diff
@@ -47,7 +47,9 @@ def _compute_scaled_data(self):
             return
 
         Y = data.Y if data.Y.ndim == 2 else np.atleast_2d(data.Y).T
-        self.original_data = np.hstack((data.X, Y)).T
+        self.original_data = np.hstack((data.X if "csr_matrix" not in str(type(data.X))
```

Comment:
To check the type it's better to use `issinstance` than checking if string is inside type's string representation. However, in this case checking for `csr_matrix` is not sufficient since this doesn't work for other sparse matrix formats (like `csc` or `dok`).

The best way to check for sparsity is to use scipy's `issparse` For example:
```
import scipy as sp
sp.issparse(data.X)
```